### DISCUSSION TITLE
How do you get your iOS data on the simulator?

### DISCUSSION THREAD

**Discussion by scottschindler** [agencyenterprise/react-native-health#149] [Show and tell] [answered]
Hi,

I'm using the iPhone simulator to run the app because I can't use Expo Go. Is there a way to import my health data into the simulator?

Also, do you need a developer account to run the app on my phone through Xcode?

Thank you,
Scott

**Comment by mgfuller333** [comment, marked as answer]
Hey you can go into the health app on the simulator and manually put data in there. Be careful about the data as it can be pretty specific for each data type
